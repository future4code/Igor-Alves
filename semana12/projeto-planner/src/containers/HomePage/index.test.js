import React from 'react';
import renderer from 'react-test-renderer';
import { HomePage } from './index'

it('Snapshot HomePage', ()=> {
    const tree = renderer
    .create(<HomePage tasks={[]} getAllTasks={()=> {}}></HomePage>)
    .toJSON();
    expect(tree).toMatchSnapshot()
})