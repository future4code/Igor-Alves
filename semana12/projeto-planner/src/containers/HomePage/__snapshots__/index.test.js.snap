// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot HomePage 1`] = `
<div
  className="rs-container"
>
  <div
    className="sc-bxivhb hdIgzJ"
  >
    <form
      className="rs-form rs-form-inline rs-form-fixed-width"
      onSubmit={[Function]}
    >
      <input
        className="sc-ifAKCX kcwBHj"
        name="text"
        onChange={[Function]}
        placeholder="Digite uma tarefa"
        required={true}
      />
      <select
        className="sc-EHOje gzokLz"
        name="day"
        onChange={[Function]}
        required={true}
      >
        <option
          disabled={true}
          selected={true}
          value=""
        >
          Dia da Semana
        </option>
        <option
          value="Segunda"
        >
          Segunda-Feira
        </option>
        <option
          value="Terça"
        >
          Terça-Feira
        </option>
        <option
          value="Quarta"
        >
          Quarta-Feira
        </option>
        <option
          value="Quinta"
        >
          Quinta-Feira
        </option>
        <option
          value="Sexta"
        >
          Sexta-Feira
        </option>
        <option
          value="Sábado"
        >
          Sábado
        </option>
        <option
          value="Domingo"
        >
          Domingo
        </option>
      </select>
      <button
        className="rs-btn rs-btn-primary rs-btn-md"
        onClick={[Function]}
        type="button"
      >
        Criar
        <span
          className="rs-ripple-pond"
        >
          <span
            className="rs-ripple"
            style={Object {}}
          />
        </span>
      </button>
    </form>
  </div>
  <div
    className="sc-htpNat jiqWYi"
  >
    <div
      className="sc-bwzfXH dXshDT"
    >
      <h3
        className="sc-bdVaJa hoqfrS"
      >
        Segunda
      </h3>
    </div>
    <div
      className="sc-bwzfXH dXshDT"
    >
      <h3
        className="sc-bdVaJa hoqfrS"
      >
        Terça
      </h3>
    </div>
    <div
      className="sc-bwzfXH dXshDT"
    >
      <h3
        className="sc-bdVaJa hoqfrS"
      >
        Quarta
      </h3>
    </div>
    <div
      className="sc-bwzfXH dXshDT"
    >
      <h3
        className="sc-bdVaJa hoqfrS"
      >
        Quinta
      </h3>
    </div>
    <div
      className="sc-bwzfXH dXshDT"
    >
      <h3
        className="sc-bdVaJa hoqfrS"
      >
        Sexta
      </h3>
    </div>
    <div
      className="sc-bwzfXH dXshDT"
    >
      <h3
        className="sc-bdVaJa hoqfrS"
      >
        Sábado
      </h3>
    </div>
    <div
      className="sc-bwzfXH dXshDT"
    >
      <h3
        className="sc-bdVaJa hoqfrS"
      >
        Domingo
      </h3>
    </div>
  </div>
</div>
`;
